FEAT: Add to Grab intent gag and blind function

LoneAsket opened the pull request:
## Описание
Добавляет синему захвату возможность при нацеливаний на рот заткнуть рот, как если бы он был заклеен скотчем.
Добавляет синему захвату возможность при нацеливаний на глаза закрыть человеку глаза как если бы ему закрыли глаза.

## Ссылка на предложение/Причина создания ПР
https://discord.com/channels/617003227182792704/755125334097133628/1146829422092103840https://discord.com/channels/617003227182792704/755125334097133628/1146829422092103840

LoneAsket committed:
FEAT: Add to Grab intent gag and blind function

Vladisvell commented on code/modules/mob/mob_grab.dm:
```diff
@@ -253,8 +264,20 @@
+			switch(hit_zone)
+				if("mouth")
+					if(!affecting.wear_mask)
+						assailant.visible_message("<span class='warning'>[assailant] закрыл[genderize_ru(assailant.gender,"","а","о","и")] рот [affecting]</span>")
+					else
+						assailant.visible_message("<span class='warning'>[assailant] схватил[genderize_ru(assailant.gender,"","а","о","и")] рот [affecting], но на нем маска!</span>")
+				if("eyes")
+					assailant.visible_message("<span class='warning'>[assailant] рукой закрыл[genderize_ru(assailant.gender,"","а","о","и")] глаза [affecting]</span>")
+				else
+					assailant.visible_message("<span class='warning'>[assailant] агрессивно схватил[genderize_ru(assailant.gender,"","а","о","и")] [affecting] (за руки)!</span>")
```
```suggestion
						assailant.visible_message(span_warning("[assailant] закрыл[genderize_ru(assailant.gender,"","а","о","и")] рот [affecting]"))
					else
						assailant.visible_message(span_warning("[assailant] схватил[genderize_ru(assailant.gender,"","а","о","и")] рот [affecting], но на нем маска!"))
				if("eyes")
					assailant.visible_message(span_warning("[assailant] рукой закрыл[genderize_ru(assailant.gender,"","а","о","и")] глаза [affecting]"))
				else
					assailant.visible_message(span_warning("[assailant] агрессивно схватил[genderize_ru(assailant.gender,"","а","о","и")] [affecting] (за руки)!"))
```

Vladisvell commented on code/modules/mob/living/say.dm:
```diff
@@ -260,6 +260,13 @@ GLOBAL_LIST_EMPTY(channel_to_radio_key)
 
+	if(!wear_mask)
+		for(var/obj/item/grab/grab in grabbed_by)
+			if(grab.assailant.zone_selected == "mouth" && grab.state == GRAB_AGGRESSIVE)
```
```suggestion
			if(grab.assailant.zone_selected == BODY_ZONE_PRECISE_MOUTH && grab.state == GRAB_AGGRESSIVE)
```

Vladisvell commented on code/modules/mob/mob_grab.dm:
```diff
@@ -136,6 +136,17 @@
 			hud.icon_state = "!reinforce"
 
+	if(state == GRAB_AGGRESSIVE)
+		if(!HAS_TRAIT(assailant, TRAIT_PACIFISM) && !GLOB.pacifism_after_gt)
+			affecting.drop_r_hand()
+			affecting.drop_l_hand()
+		if(ishuman(affecting))
```
Возможно стоит добавить ретурн в случае, если assailant таки пацифист

LoneAsket replied:
Не думаю, что стоит это делать, всё переносил строго с оригинала, добавление ретурна не в том месте может только сломать всю систему к чертям. (Бесконечные грабы, отказ работы граба и прочее)

Vladisvell reviewed (commented):
Нормас, пару стилистических правок и можно в прод

LoneAsket committed:
Update code/modules/mob/living/say.dm

Замена значения на дефайн

Co-authored-by: Vladisvell <73733747+Vladisvell@users.noreply.github.com>

LoneAsket committed:
Update code/modules/mob/mob_grab.dm

Приведение спанов к правильному и стандартному виду. Читал про это, но совсем забыл.

Co-authored-by: Vladisvell <73733747+Vladisvell@users.noreply.github.com>
